assets/src/scripts/monitoring-location/index.js

```diff
@@ -30,6 +50,8 @@ const load = function() {
         }
     });
     let nodes = document.getElementsByClassName('wdfn-component');
+    loadAllGroundWaterData(nodes, store);
```

Comment:
we are loading groundwater level data here because it is needed for both the hydrograph and dv-hydrograph components. It is only loaded if at least one of those components is present in the DOM.